project/RamlGeneratorPlugin.scala

```diff
@@ -13,7 +13,10 @@ object RamlGeneratorPlugin extends AutoPlugin {
   }
   import autoImport._
   override lazy val projectSettings = inConfig(Compile)(Seq(
-    ramlFiles := Seq(baseDirectory.value / "docs" / "docs" / "rest-api" / "public" / "api" / "v2" / "pods.raml"),
+    ramlFiles := Seq(
+      baseDirectory.value / "docs" / "docs" / "rest-api" / "public" / "api" / "v2" / "pods.raml",
+      baseDirectory.value / "docs" / "docs" / "rest-api" / "public" / "api" / "v2" / "queue.raml"
```

Comment:
not sure this is really necessary, but I was using it for validation and to test the limits of codegen